src/Nethermind/Nethermind.Core/Metric/IMetricObserver.cs

```diff
@@ -6,4 +6,5 @@ namespace Nethermind.Core.Metric;
 public interface IMetricObserver
 {
     public void Observe(IMetricLabels labels, double value);
+    public void Observe(double value);
```

Comment:
Can both methods be collapsed into:
```suggestion
    public void Observe(double value, IMetricLabels? labels = null);
```